doc/ale-cs.txt

```diff
@@ -0,0 +1,75 @@
+===============================================================================
+ALE C# Integration                                             *ale-cs-options*
+
+
+===============================================================================
+mcs                 																		           *ale-cs-mcs*
+
+  The mcs linter calls the mono mcs compiler setting the --parse and -unsafe
+  flags.
+
+g:ale_cs_mcs_options                                     *g:ale_cs_mcs_options*
+                                                         *b:ale_cs_mcs_options*
+
+  Type: String
+  Default: `''`
+
+  This variable can be changed to modify flags given to mcs. The options
+  --parse and -unsafe are implicitly set.
+
+
+===============================================================================
+mcsc                                                              *ale-cs-mcsc*
+
+	The mcsc linter uses the mono mcs compiler to generate a temporary module
+  target file (-t:module) including all '*.cs' files contained in the
+	directory by specified by |g:ale_cs_mcsc_source| or |b:ale_cs_mcsc_source|
+  variable and all sub directories. Currently none can be excluded from
+  linting. It uses the assembly directories as specified by 
+	|g:ale_cs_mcsc_assembly_path| or |b:ale_cs_mcsc_assembly_path| and selects
+  the assembly files specified by |g:ale_cs_mcsc_assemblies| or 
+  |b:ale_cs_mcsc_assemblies|. The mcs -unsafe option is set implicitly and has
+  not to be added using |g:ale_cs_mcsc_options| or |b:ale_cs_mcsc_options|
+  variable.
```

Comment:
The formatting of the text is very strange here. It looks like things are randomly indented. I find it hard to read.